### PR TITLE
Fix for #588, set location from ProviderConfig for a newResource to h…

### PR DESCRIPTION
**Pull request by majst01**
…ave the packet facility set which is a required parameter

### CODE REVIEW

File: cloud/packet/public/resources/device.go [hunk 1]
```diff
@@ -46,6 +46,14 @@ type Device struct {
 	ProjectID        string
 }
 
+func (r *Device) String() string {
```
Comment (xmudrii) [inline cloud/packet/public/resources/device.go]: Where's this function being used and what exactly it does?

The PR looks good, thanks for the contribution! I would just like to clarify this before merging.

Comment (majst01) [reply cloud/packet/public/resources/device.go]: Without the Stringer debug log does not clearly show what value each property of Device has, i can remove if you want, but it makes logs a bit more readable here.

Comment (xmudrii) [reply cloud/packet/public/resources/device.go]: I think it's okay to stay for now.

**Comment by xmudrii** [comment]
LGTM. Thank you for the contribution!

**Pull request merged**